src/cli/utils/sharepoint_test.go

```diff
@@ -17,6 +17,8 @@ func TestSharePointUtilsSuite(t *testing.T) {
 	suite.Run(t, new(SharePointUtilsSuite))
 }
 
+// Tests selector build for SharePoint properly
+// differentiates between the 3 categories: Pages, Libraries and Lists CLI
 func (suite *SharePointUtilsSuite) TestIncludeSharePointRestoreDataSelectors() {
```

Comment:
This test could use some cases that includes Pages.  (It could also use some cases for Lists, but that can be another PR).

Reply:
There are some scattered Lists tests within

Reply:
Resolved within [e441252](https://github.com/alcionai/corso/pull/2111/commits/e441252f06d2b2cbc588f09d081135177e949034)